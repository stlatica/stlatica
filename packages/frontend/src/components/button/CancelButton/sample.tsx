import React from "react";

import { CancelButton } from "./CancelButton";

export const Sample1 = () => {
  return (
    <div>
      <CancelButton>Cancel Button</CancelButton>
    </div>
  );
};
