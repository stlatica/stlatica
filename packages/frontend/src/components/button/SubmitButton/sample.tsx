import React from "react";

import { SubmitButton } from "./SubmitButton";

export const Sample1 = () => {
  return <SubmitButton>Submit Button</SubmitButton>;
};
