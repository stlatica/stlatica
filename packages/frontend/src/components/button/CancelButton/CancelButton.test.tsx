import React from "react";
import { render } from "@testing-library/react";
import { Sample1 } from "./sample";
import { describe, expect, test } from "vitest";

test("snapshot", () => {
  const { asFragment } = render(<Sample1 />);
  expect(asFragment()).toMatchSnapshot();
});
