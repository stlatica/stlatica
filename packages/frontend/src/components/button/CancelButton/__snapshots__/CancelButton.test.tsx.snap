// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot > snapshot 1`] = `
<DocumentFragment>
  <div>
    <button
      class="text-gray-900 bg-white border border-gray-300 focus:outline-none hover:bg-gray-100 focus:ring-4 focus:ring-gray-200 font-medium rounded-lg text-sm px-5 py-2.5 mr-2 mb-2 dark:bg-gray-800 dark:text-white dark:border-gray-600 dark:hover:bg-gray-700 dark:hover:border-gray-600 undefined"
      type="button"
    >
      Cancel Button
    </button>
  </div>
</DocumentFragment>
`;
